src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -37,14 +37,16 @@ public Security()
 
         internal Security(InstrumentationGateway instrumentationGateway = null, IPowerWaf powerWaf = null)
         {
+            var found = bool.TryParse(Environment.GetEnvironmentVariable("DD_ENABLE_SECURITY"), out var enabled);
```

Comment:
We have a (ridiculous long but optimized) `String.ToBoolean()` extension method which allows for different truthy values (`true`, `T`, `yes`, `Y`, `1`). We try to use it for all boolean configuration.

https://github.com/DataDog/dd-trace-dotnet/blob/b68cdfef714b378cd61ced1fa2894e0e215d90a9/src/Datadog.Trace/ExtensionMethods/StringExtensions.cs#L37

Reply:
Is `DD_ENABLE_SECURITY` the final name for this across languages? In APM most "enabled" settings follow the `DD_{FEATURE}_ENABLED` pattern.

Reply:
It is not the final version, the final version is DD_APPSEC_ENABLED. I logged this jira to remind me to change it, https://datadoghq.atlassian.net/browse/APPSEC-747, buy I'll do it now, as you mention it.

Reply:
Also use the correct boolean converter now.